### DISCUSSION TITLE
How to perform launch_menu action when default domain set to ssh_domain?

### DISCUSSION THREAD

**Discussion by dhruvinsh** [wezterm/wezterm#5421] [Q&A] [answered]
### What Operating System(s) are you running on?

Windows

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

I am using Wezterm on windows 11, minimal config looks like below. Now when I fire up the Wezterm I can straight go to WSL SSH session and that is perfect, I can duplicate the pane (`CurrentPaneDomain`) and split the pane (`CurrentPaneDomain`) as well, 

Now I want to use launch menu to fire up the PowerShell, but that doesn't work, And I need to a help to figure out what I am missing here? It is something around domain but not sure what?

```lua
return {
  ssh_domains = {
    {
      name = "wsl:ssh",
      remote_address = "192.168.4.174",
      username = "ds",
      multiplexing = "None",
      default_prog = { "zsh" },
      assume_shell = "Posix",
    },
  }
  launch_menu = {
    { label = "WSL", args = { "wsl", "--cd", "~" } },
    { label = "PowerShell Core", args = { "pwsh" } },
    { label = "Command Prompt", args = { "cmd" } },
    { label = "Nushell", args = { "nu" } },
  }
  default_domain = "wsl:ssh" 
}
```

**Comment by wez** [comment, marked as answer]
for your pwsh to run on the windows side, you want to run it in the `local` domain:

```
{ label = "PowerShell Core", args = { "pwsh" }, domain = { DomainName = 'local' }},
```

**Reply by dhruvinsh** [reply]
Yes, that was the fix, Thank you some much.
